src/utils/accessibility/handleFirstTab.ts

```diff
@@ -0,0 +1,11 @@
+import handleMouseDownOnce from './handleMouseDownOnce'
```

Comment:
this is still not fixed @nisarhassan12 ... I hope, there should be some other way to deal with this than just splitting files

Reply:
@nisarhassan12 ping, no other solutions here? :) @rogermparent hey, do you know if there is a better way to structure this?

Reply:
sorry! @shcheklein I have not found other ways than either splitting the file or disabling eslint for the line about which it was complaining. Having these functions the way they are won't cause a problem in the browser at all i.e because of the time when they are executed
both of them will be defined for sure.

![image](https://user-images.githubusercontent.com/46004116/80293829-8bd66700-877c-11ea-80ce-a5486b122396.png)

Reply:
@nisarhassan12 I think eslint disable is better if there are no other options for functions, let's go with it.